src/data/seeders/submission.js

```diff
@@ -17,6 +19,32 @@ export const up = async () => {
     totalCost: 100000,
   });
 
+  const reviewers = await User.findAll({
+    isReviewer: true,
+  });
+
+  const comments = [
+    'Complete non-sense',
+    "Best submission I've seen",
+    null,
+    null,
+  ];
+  const answers = ['accepted', 'rejected', 'accepted', 'rejected'];
+
+  await Vote.bulkCreate(
+    reviewers.slice(1, 5).map((reviewer, i) => {
+      const comment = comments[i];
+      const result = answers[i];
+
+      return {
+        userId: reviewer.id,
+        comment,
+        result,
+        submissionId: submission.id,
+      };
+    })
```

Comment:
We could also do

```js
reviewers.slice(1, 5).map((reviewer, i) => ({
  userId: reviewer.id,
  comment: comments[i],
  result: answers[i],
  submissionId: submission.id,
})
```

The intermediate variables seem pretty redundant. Not important tho